src/components/TezosOperationView.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { getOperation } from '../Conseil';
+
+interface TezosOperationProps {
+    network: string;
+    id: string;
+}
+
+interface TezosOperationState {
+    data: object;
+}
+
+export class TezosOperationView extends React.Component<TezosOperationProps, TezosOperationState> {
+
+    constructor(props: TezosOperationProps) {
+        super(props);
+        this.state = {data: {}};
+        this.refreshData = this.refreshData.bind(this);
+        this.refreshData(props);
+    }
+
+    refreshData(props: TezosOperationProps) {
```

Comment:
Higher level thought is that all the async stuff should happen in your redux pipeline and not in the component. I like [redux thunk](https://github.com/reduxjs/redux-thunk). 

Additionally, none of these async calls have a `.catch`, it looks like mostly you are checking if data exists and if not show some text saying it is loading so that's fine, but you might want to let the user know if the call fails?

Reply:
Problem with moving this to the Redux pipeline is it makes the state a lot more complicated. By making the two expansion a local behavior, we keep a simple global state. 

The async stuff should definitely have a catch. We can fix this as part of #18.

Reply:
👍